src/dmd/delegatize.d

```diff
@@ -86,6 +86,26 @@ private void lambdaSetParent(Expression e, FuncDeclaration fd)
         alias visit = typeof(super).visit;
         FuncDeclaration fd;
 
+        private void setParent(Dsymbol s)
+        {
+            VarDeclaration vd = s.isVarDeclaration();
+            FuncDeclaration pfd = s.parent ? s.parent.isFuncDeclaration() : null;            if (s.isVarDeclaration() && s.parent.isFuncDeclaration())
```

Comment:
little problem with the editor commands I suppose ? (check the end of this line)

Reply:
Ah, sorry. Thanks.